app/javascript/components/topology_toolbar.jsx

```diff
@@ -43,12 +43,10 @@ const TopologyToolbar = () => (
             <button type="button" className="clear" aria-hidden="true" onClick={resetSearchClick}>
               <span className="pficon pficon-close" />
             </button>
-          </div>
-        </div>
-        <div className="form-group search-button">
-          <button type="button" className="btn btn-default search-topology-button" onClick={searchClick}>
+            <button type="button" className="btn btn-default search-topology-button" onClick={searchClick}>
             <span className="fa fa-search" />
           </button>
+          </div>
```

Comment:
Either the indentation is broken here, or the number of opened/closed divs no longer matches...

(The button and the div should not be on the same indent level, right?)